deploy/upgrade-creds.sh

```diff
@@ -15,10 +15,6 @@
 
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
```

Comment:
Should these lines be removed? The files no longer exist in the repo but the script is used to upgrade from a previous version where they did.

Reply:
I see no problem with removing these lines.  Those defaults really no longer make any sense since as you said, those files no longer exist at those locations (or at all, for that matter).  Therefore, any pre-v4.1.0 holdouts (do they even still exist :thinking:) can just pass in the location of these files via params, as supported by the script.